variables.tf

```diff
@@ -10,7 +10,7 @@ variable "name" {
  }
 
  variable "instance_type" {
-  default =  "t2.small"
```

Comment:
why default Dan?

Reply:
sorry I missed that . will fix it straight away

Reply:
Looks great. Proceed

Reply:
Thank you for the merge